<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fractal Tool </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="viewOriginal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="viewProcessed">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFlush">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Flush changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDefaults">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore defaults</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editBrightness">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>460</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelBrightness">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit brightness:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>440</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>440</y>
      <width>20</width>
      <height>111</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>460</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Change image size (save as):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>500</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>500</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editWidth">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>500</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editHeight">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>500</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Commands</string>
    </property>
    <addaction name="actionConvert_to_GRAY"/>
    <addaction name="actionConvert_to_B_W"/>
    <addaction name="actionCreate_grid"/>
    <addaction name="actionTEST"/>
    <addaction name="actionFace"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuConfiguration"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOriginal_Image">
   <property name="text">
    <string>Original image</string>
   </property>
  </action>
  <action name="actionImage_after_processing">
   <property name="text">
    <string>Image after processing</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConvert_to_GRAY">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Convert to GRAY</string>
   </property>
  </action>
  <action name="actionJPG">
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="actionBMP">
   <property name="text">
    <string>BMP</string>
   </property>
  </action>
  <action name="actionSave_As_2">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionCreate_grid">
   <property name="text">
    <string>Create grid</string>
   </property>
  </action>
  <action name="actionTEST">
   <property name="text">
    <string>TEST</string>
   </property>
  </action>
  <action name="actionFace">
   <property name="text">
    <string>Face Detection</string>
   </property>
  </action>
  <action name="actionConvert_to_B_W">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Convert to B&amp;W</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
